plugins/chat/assets/javascripts/discourse/components/chat-thread.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="chat-thread" data-id={{this.thread.id}}>
+  <div class="chat-thread__header">
+    <div class="chat-thread__info">
+      <div class="chat-thread__title">
+        <h2>{{this.title}}</h2>
+
+        <LinkTo
+          class="chat-thread__close"
+          @route="chat.channel"
+          @models={{this.chat.activeChannel.routeModels}}
+        >
+          {{d-icon "times"}}
+        </LinkTo>
+      </div>
+
+      <p class="chat-thread__om">
+        {{replace-emoji this.thread.original_message.excerpt}}
+      </p>
+
+      <div class="chat-thread__op">
```

Comment:
what is op here?

Reply:
original poster

Reply:
it's a little bit confusing, given we have the very well-known `op` for `original post` I understand the context is different but similar enough to be confusing.

how about `chat-thread__omp` ? original message poster? Not super elegant, but it's uniq, concise and consistent

Reply:
I think `chat-thread__omu` would be better, since that matches `original_message_user`

Reply:
works for me too, up to you :+1: